native/storybook/storybook.js

```diff
@@ -33,13 +39,34 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     padding: spacingBase,
     width: '100%',
+    marginTop: spacingLg,
+  },
+  rtlButton: {
+    justifyContent: 'flex-end',
+    marginVertical: spacingBase,
+    width: spacingXxl * 3,
   },
 });
+const toggleRTL = () => {
+  I18nManager.forceRTL(!I18nManager.isRTL);
+  RNRestart.Restart();
+};
 
 const CenterDecorator = getStory => (
   <View style={styles.centered}>{getStory()}</View>
 );
-
+const RTLDecorator = getStory => (
```

Comment:
This is super neat, but don't we have to strings in an RTL language for this to be truly useful too?

Reply:
yes and no, because of RN's limitations (see https://facebook.github.io/react-native/blog/2016/08/19/right-to-left-support-for-react-native-apps.html#limitations-and-future-plan) this is probably the best we can get right now

Reply:
Okay, maybe we should explore different langauge bundles for iOS then